tests/test_aggregation_embeddings.py

```diff
@@ -12,7 +12,7 @@
 from embeddings.embedding.static.word2vec import KGR10Word2VecConfig
 
 
-@pytest.fixture
+@pytest.fixture(scope="module")
```

Comment:
All fixtures' scopes were changed to module level? What is the reason for this change? If I am correct, this fixture will be overwritten when defined in another file.